src/caustics/tests.py

```diff
@@ -10,14 +10,24 @@
 
 __all__ = ["test"]
 
+# Pseudo device as mentioned in https://github.com/pytorch/pytorch/issues/61654
+# using this device, no actual computation is done, but will check
+# for where the tensors are located
+META_DEVICE = torch.device("meta")
 
-def _test_simulator_runs():
+TORCH_DEVICE = (
```

Comment:
I've usually seen it written as just `DEVICE` not `TORCH_DEVICE`. We could change that just to keep with convention.